static/css/restyle/restyle.less

```diff
@@ -1415,6 +1419,10 @@ h3.author .transfer-ownership {
   padding: 30px 0;
   width: 100vw;
 
+  .html-rtl& {
```

Comment:
`&` is just a reference to the parent selector.  For the second fix here the parent has the class `.restyle` which is also on the body like `.html-rtl` is, hence no space.